lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -345,6 +346,7 @@
 }
 </script>
 
+<!-- PASS: not in header, does not block rendering -->
```

Comment:
oh yeah actually could you comment that it's there to polyfill promise and make sure we don't break :)